### DISCUSSION TITLE
Got (tag.text || "") .trim is not a function error on docgen when using React.HTMLAttributes<HTMLButtonElement>

### DISCUSSION THREAD

**Discussion by ddsuhaimi** [storybookjs/storybook#15106] [Help] [answered]
Here's the component
```
import React, { FC } from "react";
export interface ButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
    className?: string;
    disabled?: boolean;
    active?: boolean;
    primary?: boolean;
    onClick?: () => void;
}

export const Button: FC<ButtonProps> = (props) => {
    return <button {...props}>tes</button>;
};

```
![image](https://user-images.githubusercontent.com/30659198/120226604-b93fef00-c271-11eb-870c-e67d8d412905.png)

I did the exact same thing in codesandbox but it works:
https://codesandbox.io/s/autumn-rain-ngtsp?file=/src/Test.tsx

As soon as I remove extends React.HTMLAttributes<HTMLButtonElement> everything works fine.

**Comment by jorisw** [comment, marked as answer]
Related issue: https://github.com/storybookjs/storybook/issues/15067

**Reply by shilman** [reply]
Dupe to #15067
